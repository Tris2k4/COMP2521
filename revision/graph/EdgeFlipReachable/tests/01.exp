Enter number of vertices: Enter number of edges: Enter edges in the form v-w: 
Graph: nV = 5
Edges:
 0: 0-1
 1: 1-3 1-4
 2: 2-1 2-3
 3:
 4:

Enter the source vertex: Reachable from vertex 0:
0
1
2
3
4
Enter the source vertex: Reachable from vertex 1:
0
1
2
3
4
Enter the source vertex: Reachable from vertex 2:
0
1
2
3
4
Enter the source vertex: Reachable from vertex 3:
3
Enter the source vertex: Reachable from vertex 4:
4
Enter the source vertex: 