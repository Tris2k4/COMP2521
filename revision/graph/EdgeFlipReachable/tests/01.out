Enter number of vertices: Enter number of edges: Enter edges in the form v-w: 
Graph: nV = 5
Edges:
 0: 0-1
 1: 1-3 1-4
 2: 2-1 2-3
 3:
 4:

Enter the source vertex: Reachable from vertex 0:
0 2 1 
Enter the source vertex: Reachable from vertex 1:
2 1 
Enter the source vertex: Reachable from vertex 2:
2 1 1 
Enter the source vertex: Reachable from vertex 3:
3
Enter the source vertex: Reachable from vertex 4:
4
Enter the source vertex: 